platforms/hyperledger-besu/configuration/roles/delete/helm_releases/tasks/nested.yaml

```diff
@@ -0,0 +1,7 @@
+# Storing release name and its namespace in an array
+- name: Nested loop for helm releases
+  set_fact:
+    helm_result={{ helm_result | default([]) + [{'release_name':helm_resource.spec.releaseName, 'ns':helm_resource.metadata.namespace}] }}
+  loop: "{{ resources.resources | list }}"
```

Comment:
I don't think this is tested.
There is no variable named `resources` here.
Also the variable `helm_releases_raw` created in main.yaml doesnt seem to be used here.

Please check and verify this.

Reply:
Hi @jagpreetsinghsasan 

Yes, that part of the code was missing. Thank you for letting me know about that. Now, the code has been updated and tested successfully.